README.md

```diff
@@ -328,5 +329,7 @@ _([Nix as observed by LORRI on 2015-07-13](https://www.nasa.gov/newhorizons/lorr
 [home-manager-service]: https://rycee.gitlab.io/home-manager/options.html#opt-services.lorri.enable
 [home-manager]: https://rycee.gitlab.io/home-manager/
 [lorri-blog-post]: https://www.tweag.io/posts/2019-03-28-introducing-lorri.html
+[nix-darwin]: https://daiderd.com/nix-darwin/
+[nix-darwin-service]: https://github.com/LnL7/nix-darwin/blob/master/modules/services/lorri.nix
```

Comment:
Once the `nix-darwin` published documentation has been updated we can point to it in a similar manner to how `nixos-service` and `home-manager-service` link to that documentation.